tests/version_tools/test_version_tools.py

```diff
@@ -0,0 +1,7 @@
+from notifications_utils.version_tools import get_file_contents_from_github, get_remote_version
+
+
+def test_get_remote_version():
```

Comment:
Not 100% sure we need a test for this but if we’re going to have one it shouldn’t go out to the internet.

Otherwise we:
- cannot run the tests offline 
- will get rate limited by Github’s API when running the tests multiple times

Either:
```suggestion
def test_get_remote_version():
    mock_requests_get = mocker.patch("notifications_utils.version_tools.requests.get", return_value=…)
```

Or:
```suggestion
def test_get_remote_version():
    mock_get_file_content_from_github = mocker.patch("get_file_contents_from_github", return_value='__version__ = "1.2.3"')
```